.binstar.yml

```diff
@@ -2,21 +2,17 @@ package: nb_anacondacloud
 
 platform:
   - linux-64
-  - linux-32
-  - osx-64
-  - win-64
-  - win-32
+  # - linux-32
+  # - osx-64
+  # - win-64
+  # - win-32
```

Comment:
we need build in all the platforms... if running the test is difficult in win, we could potentially deactivate the testing on win until the problems are solved, but still, we should build in all the platforms, I think...